recipes/pymablock/meta.yaml

```
@@ -0,0 +1,52 @@
+{% set name = "pymablock" %}
+{% set version = "1.0.0.post0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 880da5a73b5c7b8e275ac1ace8835a8e632c9632fbcca400b859c6e4ccc11f20
+
+build:
+  script: {{ PYTHON }} -m pip install . --no-deps -vv
+  skip: true  # [py<39]
+  number: 0
+
+requirements:
+  host:
+    - python
+    - pip
+    - hatchling
+    - hatch-vcs
+  run:
+    - python
+    - numpy >=1.23
+    - scipy >=1.8
+    - sympy >=1.10
+    - packaging >=22.0
+    # Optional dependencies
+    - kwant >=1.4  # [not win]
```

I could not find this in the package metadata but if you need this you can still keep it noarch and do:

```suggestion
    # Optional dependencies
    - kwant >=1.4  # [not win]
    - __win      # [win]
    - __unix     # [not win]
```

Those last two lines will ensure the package is "noarch" for Python but "semi-arch" for the extra dependency. Note that the linter will fail but we are updating it to accept this syntax.

---

Thanks, this seems like what we need. The package is not in metadata is due to a binary dependency only packaged via conda and not on windows.